src/styles/blocks/card.scss

```diff
@@ -0,0 +1,179 @@
+.cards {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  padding: 50px 40px;
+  margin: 0 auto;
```

Comment:
- Check it everywhere, please 
<img width="989" alt="image" src="https://user-images.githubusercontent.com/99403198/222898174-077ca58c-7cf2-45ec-954b-287e615c87dd.png">